addon/components/search-dropdown/template.hbs

```diff
@@ -3,7 +3,7 @@
         <div class="row">
             <div class="col-md-12">
                 <form class="input-group" {{action "search" on="submit"}}>
-                    <input id="searchPageFullBar" onchange={{action "updateQuery" value="target.value"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search">
+                    <input id="searchPageFullBar" onchange={{action "updateQuery" value="target.value"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search the OSF">
```

Comment:
It appears that `updateQuery` purely exists to set a variable to the field value whenever the field changes.

This sounds like a use case for two-way data binding, which is [set up automatically](https://guides.emberjs.com/v2.6.0/templates/input-helpers/) (no action needed) if you use the curly brace helper form:

`{{input value=query}}`

Since `query` seems to be just a placeholder value, a two-way binding might be a concise approach for this specific situation.